README.md

```diff
@@ -4,11 +4,20 @@ Tools, research, ideas and experiments, made by muze.nl
 
 ## Tools
 
-### SimplyEdit &amp; SimplyView
-
-- [SimplyEdit](https://simplyedit.io) [github](https://github.com/simplyedit/simplyedit)
-- [SimplyView](https://reference.simplyedit.io/simplyview) [github](https://github.com/simplyedit/simplyview)
-- [json-css](https://github.com/simplyedit/json-css)
+### Simply Tech Stack
```

Comment:
It might make more sense to sort things by project maturity? Noy sure...

---

#### Mature

- SimplyEdit
- SimplyView

#### Stable

- SimplyDomain
- json-css-search

#### Beta

- SimplyPresent
- jsontag jsontag-types od-jsontag
- jaqt

#### Alpha

- SimplyStore
- SimplyCode
- metrojs
- oldm
- jsfs

Reply:
My only comment would be that the `(stable)` etc. are now no longer needed behind the entries.

Reply:
done!